ibm_mq/tests/test_ibm_mq_int.py

```diff
@@ -173,3 +174,19 @@ def test_check_regex_tag(aggregator, instance_queue_regex_tag, seed_data):
 
     for metric, _ in QUEUE_METRICS:
         aggregator.assert_metric(metric, tags=tags)
+
+
+def test_parsing():
```

Comment:
```suggestion
def test_metadata_parsing():
```

Including `metadata` in the test name will make it easier to understand at a glance.